### DISCUSSION TITLE
And not all channels have a stream

### DISCUSSION THREAD

**Discussion by ghost** [iptv-org/community#486] [Show and tell]
People, if you want TV, get a cable or satellite service and don't be fooled by people like this, just like I was fooled.

**Comment by BellezaEmporium** [comment]
???

**Reply by 9123086** [reply]
@AlexIs387 
if you have ROKU device or ROKU tv, you may try IOTV channel,

[https://my.roku.com/account/add/iotv0815](url)

there are many channels are available. 
the good part is you can watch live channels from different countries world wide, although the languages might not understand, the video scenes are interesting
